lighthouse-core/lib/i18n/locales.js

```diff
@@ -39,6 +39,7 @@ const locales = {
   'de': require('./locales/de.json'), // de-AT, de-CH identical, so they fall back into de
   'el': require('./locales/el.json'),
   'en-XA': require('./locales/en-XA.json'), // psuedolocalization
+  'en-XA-local': require('./locales/en-XA-local.json'), // local psuedolocalization
```

Comment:
since we're keeping the other en-XA let's disambiguate

(also i think variants would be in the middle of the locale tag... like en-local-XA)

but anyway.. lets just call this `en-XL`  L for local. ;)

Reply:
sgtm!